rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```
@@ -2942,6 +2942,21 @@ public String loginOrRenewSession(@HeaderParam("sessionid") String sessionId,
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override
+    @GET
+    @Path("logins/sessionid/{sessionId}")
+    @Produces("application/json")
+    public String getLoginFromSessionId(@PathParam("sessionId") String sessionId)
+            throws SchedulerRestException, LoginException, NotConnectedRestException {
+        if (sessionId != null && sessionStore.exists(sessionId)) {
+            return sessionStore.get(sessionId).getUserName();
+        }
+        return "";
```

Is it not better to throw an exception in that case?